pkg/operator/controller/nodeobservability/controller.go

```diff
@@ -19,6 +19,11 @@ package nodeobservabilitycontroller
 import (
 	"context"
 	"fmt"
+	"strings"
```

Comment:
Let's preserve the forrmatting order. It goes:

1. [k8s.io/](http://k8s.io/)
2. [github.com/openshift/api](http://github.com/openshift/api)
3. 3rd-party libraries (github, sigs, etc)
4. our own module.

Reply:
Addressed